pkg/api/controller_test.go

```diff
@@ -73,31 +72,23 @@ import (
 )
 
 const (
-	username               = "test"
-	htpasswdUsername       = "htpasswduser"
-	passphrase             = "test"
-	group                  = "test"
-	repo                   = "test"
 	ServerCert             = "../../test/data/server.cert"
 	ServerKey              = "../../test/data/server.key"
 	CACert                 = "../../test/data/ca.crt"
-	AuthorizedNamespace    = "everyone/isallowed"
 	UnauthorizedNamespace  = "fortknox/notallowed"
-	ALICE                  = "alice"
 	AuthorizationNamespace = "authz/image"
-	AuthorizationAllRepos  = "**"
+	LDAPAddress            = "127.0.0.1"
 )
 
-func getCredString(username, password string) string {
-	hash, err := bcrypt.GenerateFromPassword([]byte(password), 10)
-	if err != nil {
-		panic(err)
-	}
-
-	usernameAndHash := fmt.Sprintf("%s:%s", username, string(hash))
-
-	return usernameAndHash
-}
+var (
+	username         = test.GenerateRandomString() //nolint: gochecknoglobals
```

Comment:
Are all these variables used?

Reply:
Yes. There are used by LDAP related tests that rely on some common username between different tests.
In a different PR it will make sense to separate those tests into some ldap_tests.go